lib/reek/context/attribute_context.rb

```diff
@@ -15,6 +19,14 @@ def full_comment
         send_expression.full_comment || ''
       end
 
+      def instance_method?
+        true
+      end
+
+      def apply_current_visibility(current_visibility)
```

Comment:
Since this method now basically boiled down to a setter, wouldn't it make moar sense to just use "attr_writer"?
Because right now the name is so pretentious, that it implies that there is a lot more happening than just a setter.